src/pages/journal/journal-components/journal-components.module.ts

```diff
@@ -1,32 +0,0 @@
-import { NgModule } from '@angular/core';
```

Comment:
I see you've pulled this module up into `journal.module.ts`. Is this due to some technical reason? Or is it just that having a component module per page not the preferred pattern?

Reply:
Both :)

 In the how to lay out stuff meeting at the beginning we agreed to put all the components specific to a page in the page component.

Also when putting in the dynamic stuff in this PR we needed to put JournalTestSlot as an entry component into the Journal Page module, having the component in both modules started to bring down the walls between the 2 modules